src/platform/macos/package_handler.h

```diff
@@ -0,0 +1,29 @@
+// SPDX-FileCopyrightText:  2025 The DOSBox Staging Team
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#ifndef DOSBOX_PLATFORM_MACOS_PACKAGE_HANDLER_H
+#define DOSBOX_PLATFORM_MACOS_PACKAGE_HANDLER_H
+
+#include <optional>
+#include <string>
+#include <vector>
+
+namespace macos {
+
+// Represents the expanded form of a .dosbox package
+struct DosBoxPackage {
```

Comment:
`MacOsDosBoxPackage`?

We use namspaces in more recent code (e.g., for CLAP plugin support), but I don't really like doing it for platform-specific stuff.